Scripts/Data/Type/Transformation/Vector3Subtractor.cs

```diff
@@ -0,0 +1,41 @@
+﻿namespace VRTK.Core.Data.Type.Transformation
+{
+    using UnityEngine;
+    using UnityEngine.Events;
+    using System;
+    using System.Linq;
+
+    /// <summary>
+    /// Processes a given <see cref="Vector3"/> and subracts all values in the collection and outputs the sum of all values.
```

Comment:
This seems to be no longer true. What about this instead:
```suggestion
    /// Subtracts a collection of <see cref="Vector3"/>s by subtracting each one from the first entry in the collection.
```

If the above won't be used there are typos to fix:
```suggestion
    /// Processes a given <see cref="Vector3"/> and subtracts all values in the collection and outputs the subtraction of all values.
```